airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py

```diff
@@ -98,10 +98,11 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         """
         records = response.json().get(self.data_field) or []
         for record in records:
-            if record.get(self.data_field):
-                yield record.get(self.data_field)
-            else:
-                yield record
+            record = record.get(self.data_field) or record
+            if self.primary_key in record and record[self.primary_key] is None:
+                # Convert "id: null" fields to "id: 0" since id is primary key and SAT checking if it not null.
+                record[self.primary_key] = 0
```

Comment:
You may just set `primary_key=None` and SAT will skip the stream from the pk validation. 
Or is it may be some pks with integer values and some are None?

Reply:
> Or is it may be some pks with integer values and some are None?

Yes this is the case, there is mixed null and integer values for activity_fields stream: https://pastebin.com/0KvSSzYp